src/i18n/nb_NO.ts

```diff
@@ -0,0 +1,863 @@
+            maxPersonnel: 4,
+            special: 'Required once you have built 8 police stations',
+            schooling: 'Police - SWAT',
+            shownSchooling: 'SWAT',
+        },
+        16: {
+            caption: 'Røykdykkerbil',
+            color: '#aa0000',
+            coins: 25,
+            credits: 11_860,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            special: 'Nødvendig etter at du har bygd 5 brannstasjoner.',
+        },
+        17: {
+            caption: 'Innsatsstøttebil',
+            color: '#791515',
+            coins: 25,
+            credits: 25_500,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            schooling: 'Krever spesialistutdanning (Skadestedsledelse) ',
+            shownSchooling: 'Innsatsstøttebil',
+            special: 'Nødvendig etter at du har bygd 13 brannstasjoner.',
+        },
+        18: {
+            caption: 'Redningsbil',
+            color: '#880000',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        19: {
+            caption: 'Dykkerbil',
+            color: '#772222',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 4,
+            maxPersonnel: 5,
+            special: 'Krever spesialistutdanning (Redningsdykker kurs)'
+        },
+        20: {
+            caption: 'Lett redningsbåt',
+            color: '#772222',
+            coins: 12,
+            credits: 6_000,
+            maxPersonnel: 0,
+            special: 'Et egnet kjøretøy trengs for å trekke tilhengeren / båthenger. (Dykkerbil, Mannskapsbil, Lett mannskapsbil)'
+        },
+        21: {
+            caption: 'Branntankbil',
+            color: '#570f0f',
+            coins: 19,
+            credits: 19_000,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+        },
+        22: {
+            caption: 'Utrykningspolitibil',
+            color: '#ad0e0e',
+            coins: 10,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            schooling: 'Politi - Utrykningspoliti',
+            shownSchooling: 'Utrykningspoliti',
+        },
+        23: {
+            caption: 'Utrykningsenhetbil',
+            color: '#ad0e0e',
+            coins: 15,
+            credits: 19_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            schooling: 'Politi - Våpentrening',
+            shownSchooling: 'Våpentrening',
+        },
+    },
+    buildings: {
+        0: {
+            caption: 'Brannstasjon',
+            color: '#bb0000',
+            coins: 30,
+            credits: 100_000,
+            extensions: [
+                {
+                    caption: 'Ambulanseutvidelse',
+                    credits: 100_000,
+                    coins: 20,
+                    duration: '7 Days',
```

Comment:
days could be translated. I saw it on more buildings.
Also other things in the building information isn't translated.
Sorry, if you are already working on it.